static/js/ducks/followup_requests.js

```diff
@@ -25,9 +27,18 @@ export const prioritizeFollowupRequests = (params = {}) =>
     params
   );
 
-messageHandler.add((actionType, payload, dispatch) => {
+messageHandler.add((actionType, payload, dispatch, getState) => {
+  const { source } = getState();
+
   if (actionType === REFRESH_FOLLOWUP_REQUESTS) {
-    dispatch(fetchFollowupRequests());
+    const loaded_obj_key = source?.internal_key;
+    if ("obj_key" in payload) {
+      if (loaded_obj_key === payload.obj_key) {
+        dispatch(sourceAction.fetchSource(source.id));
```

Comment:
If we're going to fetch the source, why not change this to `REFRESH_SOURCE` as used elsewhere?

Reply:
Yeah that seems better.